pkg/controller/nodeipam/nolegacyprovider.go

```diff
@@ -34,7 +34,7 @@ func startLegacyIPAM(
 	kubeClient clientset.Interface,
 	clusterCIDRs []*net.IPNet,
 	serviceCIDR *net.IPNet,
-	nodeCIDRMaskSize int,
+	nodeCIDRMaskSizes []int,
```

Comment:
is this safe to change?

Reply:
this function doesn't ever actually run... 🙃

Reply:
this is the fake when we are building without providers. it's just so the code compiles.
see: two lines down where if we ever runt his we panic 🙃